builder/builder-source-patch.c

```diff
@@ -254,6 +254,14 @@ builder_source_patch_extract (BuilderSource  *source,
   return TRUE;
 }
 
+static gboolean
+builder_source_patch_bundle (BuilderSource  *source,
+                             BuilderContext *context,
+                             GError        **error)
+{
+  return TRUE;
```

Comment:
What was the reasoning behind doing nothing here? That the patches are already applied/copied? I was wondering how does this work, because the bundled archives and git repos are probably unmodified/unpatched? A comment about the reasoning would be very helpful here. Thanks.